src/qt3utils/datagenerators/princeton.py

```diff
@@ -306,7 +311,7 @@ def acquire_frame(self) -> Tuple[np.ndarray, np.ndarray]:
         """
         return self.light.acquire(), self.get_wavelengths()
     
-    def acquire_step_and_glue(self, wavelength_range: Tuple[float, float]) -> Tuple[np.ndarray, np.ndarray]:
+    def acquire_step_and_glue(self, wavelength_range: List[float]) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
why was this changed from a Tuple to a List? wavelength_range has exactly two elements, right?